llama-index-core/llama_index/core/instrumentation/events/embedding.py

```diff
@@ -36,3 +36,37 @@ class EmbeddingEndEvent(BaseEvent):
     def class_name(cls) -> str:
         """Class name."""
         return "EmbeddingEndEvent"
+
+
+class SparseEmbeddingStartEvent(BaseEvent):
+    """EmbeddingStartEvent.
+
+    Args:
+        model_dict (dict): Model dictionary containing details about the embedding model.
+    """
+
+    model_config = ConfigDict(protected_namespaces=("pydantic_model_",))
+    model_dict: dict
+
+    @classmethod
+    def class_name(cls) -> str:
+        """Class name."""
+        return "EmbeddingStartEvent"
+
+
+class SparseEmbeddingEndEvent(BaseEvent):
+    """EmbeddingEndEvent.
+
+    Args:
+        chunks (List[str]): List of chunks.
+        embeddings (List[List[float]]): List of embeddings.
+
+    """
+
+    chunks: List[str]
+    embeddings: List[Dict[int, float]]
+
+    @classmethod
+    def class_name(cls) -> str:
+        """Class name."""
+        return "EmbeddingEndEvent"
```

Comment:
I think this should be `SparseEmbeddingEndEvent`

Reply:
Done!